server/logger.js

```diff
@@ -9,31 +9,15 @@ if (!fs.existsSync(logDir)) {
   fs.mkdirSync(logDir);
 }
 
-const logger = new winston.Logger({
-  transports: [
-    new winston.transports.File({
-      level: env === 'development' ? 'debug' : 'info',
-      filename: logDir.concat('/logs.log'),
-      handleExceptions: true,
-      json: true,
-      maxsize: 5242880, // 5MB
-      maxFiles: 5,
-      colorize: false
-    }),
-    new winston.transports.Console({
-      level: 'debug',
-      handleExceptions: true,
-      json: false,
-      colorize: true
-    })
-  ],
-  exitOnError: false
+const logger = new blgr();
+logger.set({
+  level: env === 'development' ? 'debug' : 'info',
+  colors: true,
+  filename: logDir.concat('/logs.log')
```

Comment:
Thoughts on this location for the logs (gets set to `__dirname/../logs` above)? This was before we were using bcfg and before there was a `prefix`. Would logs be better in `~/.bpanel` (or wherever a custom prefix is set to)?